Deploy-Samples/Terraform/arm/vi.template.json

```diff
@@ -0,0 +1,78 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "name": {
+            "type": "string",
+            "metadata": {
+                "description": "The name of the video indexer resource."
+            }
+        },
+        "managedIdentityResourceId": {
+            "type": "string",
+            "metadata": {
+                "description": "The managed identity resource Id used to grant access to the azure media service (AMS) account."
+            }
+        },
+        "mediaServiceAccountResourceId": {
+            "type": "string",
+            "metadata": {
+                "description": "The media service account resource Id. The account needs to be created prior to the creation of this template."
+            }
+        },
+        "tags": {
+            "type": "object",
+            "metadata": {
+                "description": "Tags which should be assigned to video indexer resource."
+            },
+            "defaultValue": {}
+        }
+    },
+    "resources": [
+        {
+            "type": "Microsoft.VideoIndexer/accounts",

```

Comment:
Don't you have to have the AMS itself created?

Reply:
![image](https://user-images.githubusercontent.com/512709/219072746-2e65d85e-a9fc-4ab6-98d8-2586f0ac4ae5.png)

Reply:
AMS is created within the terraform template.
since VI Does not have a terraform template , the last part of the creation in done with mix-and-match integration
of Terraform and ARM.